### PR TITLE
add exclusive touch to rubric buttons

### PR DESCRIPTION
**Pull request by bluwave**
refs: none
affects: Student
release note: none

### CODE REVIEW

File: Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift [hunk 1]
```diff
@@ -105,6 +105,7 @@ class RubricCircleView: UIView {
             }
             let button = DynamicButton(frame: CGRect(x: center.x, y: center.y, width: circleWidth, height: RubricCircleView.w))
             button.tag = i
+            button.isExclusiveTouch = true
```
Comment (armstrongnate) [inline Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift]: What does this do?

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Student

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Core** | 95.32% | 95.32% | 0%
**Student** | 90.8% | 90.8% | 0%
**Teacher** | 100% | 100% | 0%
**React Native** | 92.79% | 92.79% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift (hunk 1 above)
Comment (bluwave) [reply Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift]: Makes it so you can’t press multiple buttons at once. I think you found this one.

Comment (armstrongnate) [reply Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift]: Nice!

**Comment by armstrongnate** [comment]
Still happening for me..

IMG_9F15A89B1412-1

**Pull request merged**